templates/web/fixmystreet.com/index.html

```diff
@@ -0,0 +1,20 @@
+[% map_js = PROCESS 'front/javascript.html' %]
```

Comment:
I would prefer not to override index.html. If you have an empty footer-marketing template, does that not have basically the same effect? Given none of that is shown any more.